src/gobprepare/importers/csv_importer.py

```
@@ -17,9 +22,25 @@ class PostgresCsvImporter():
 
     def __init__(self, dst_connection, config: dict):
         self._dst_connection = dst_connection
-        self._source = config['source']
         self._destination = config['destination']
 
+        if config.get('objectstore'):
+            self._source = self._load_from_objectstore(config['objectstore'], config['source'])
+        else:
+            self._source = config['source']
+
+        # Mapping of CSV columns to database columns (default CSV columns will be used if no alternative supplied)
+        self._column_names = config.get('column_names', {})
+        self._separator = config.get('separator', ',')
```

maybe make a constant for default separator to do not hide this in the code.